src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -334,10 +333,9 @@ private void DispatchRenderFrame()
             if (elapsed > 0 && elapsed >= renderPeriod)
             {
                 _watchRender.Restart();
+                RenderTime = elapsed;
```

Comment:
In this loop we also should update `elapsed` at the end of each loop.
We should probably also do the same kind of `UpdateFrequency <= double.Epsilon` check that `DispatchUpdateFrame` does.

Reply:
this is an if statement not a loop

Reply:
Oh yeah, lol